### ISSUE TITLE
Some Questions about the code

### ISSUE THREAD

**Issue by dldl3D** [lasinger/3DVideos2Stereo#1]
Thanks your code. I use the code to generate the training data. I have some questions about it as follows:

1) How to use disp and Uncertainty? What does the Uncertainty mean?
2) How to visualize the depth image in the paper?
3) Do you select the training data manually? Some results of the disparity are very bad after filtering.

**Comment by lasinger** [comment]
1. See [get_disp_and_uncertainty.py](https://github.com/lasinger/3DVideos2Stereo/blob/master/get_disp_and_uncertainty.py) for details. Uncertainty is the difference between the forward and backward flow, hence, can be used for the left-right-consistency check and to create the mask of valid pixels. As described in the supplementary material of the paper, we mark pixels with a difference of more than 2 pixels as invalid. Furthermore, automatic filtering of bad frames is performed in [get_disp_and_uncertainty.py](https://github.com/lasinger/3DVideos2Stereo/blob/master/get_disp_and_uncertainty.py).

2. We use matplotlib to visualize disparity maps in our paper. E.g. 
`plt.imsave(filename, dispmap, cmap=plt.cm.inferno)`
In case of available ground truth, we shifted and scaled the prediction to the ground truth and used the min and max disparity values of the ground truth for the parameters vmin and vmax for both visualizations of ground truth and prediction (same also for the results of other methods in the paper). For video sequence visualization, we also fixed vmin and vmax to the minimum and maximum of the sequence, hence, all frames are scaled jointly.

3. We did not perform any manual filtering, only the automatic filtering that was described in the paper.
